### DISCUSSION TITLE
How does a complete novice learn about ai agents or multiagents

### DISCUSSION THREAD

**Discussion by zerd1y** [microsoft/autogen#3671] [Q&A]
I am a first year undergraduate student in China, and I didn't come into contact with programming until I went to university. At present, I am very interested in ai agents and multiagents, but I can only use C language and C++ at present, and I know nothing about these things that represent the future.

May I ask:

1. What should I do now? What knowledge and technology to learn?

2. What courses do you recommend?

**Comment by ekzhu** [comment]
Maybe start learning Python?

**Reply by zerd1y** [reply]
I have received your suggestion, thank you very much. I will learn relevant knowledge in my next time！！
